sample/src/main/java/org/aerogear/graphqlandroid/activities/MainActivity.kt

```diff
@@ -107,30 +134,39 @@ class MainActivity : AppCompatActivity() {
 
         Log.e(TAG, "inside update task")
 
-        val client = Utils.getApolloClient(this)?.mutate(
-            UpdateCurrentTask.builder().id(id).title(title).version(version).build()
-        )
-        client?.enqueue(object : ApolloCall.Callback<UpdateCurrentTask.Data>() {
-            override fun onFailure(e: ApolloException) {
-                Log.e("onFailure" + "updateTask", e.toString())
+        val client: ApolloMutationCall<UpdateCurrentTask.Data>?
+
+        val mutation = UpdateCurrentTask.builder().id(id).title(title).version(version).build()
+
+        client = Utils.getApolloClient(this)?.mutate(mutation)?.refetchQueries(object : OperationName {
```

Comment:
`refetchQueries()` requires an object of the type `OperationName` or `Query`.

What you've passed here looks lik the OperationDefinition, why don't you get the OPERATION_NAME from AllTaskQuery and pass it here instead?